src/test/java/com/sonyericsson/jenkins/plugins/bfa/jcasc/ConfigurationAsCodeLocalTest.java

```diff
@@ -0,0 +1,88 @@
+package com.sonyericsson.jenkins.plugins.bfa.jcasc;
+
+import com.sonyericsson.jenkins.plugins.bfa.PluginImpl;
+import com.sonyericsson.jenkins.plugins.bfa.db.LocalFileKnowledgeBase;
+import com.sonyericsson.jenkins.plugins.bfa.sod.ScanOnDemandVariables;
+import io.jenkins.plugins.casc.ConfigurationContext;
+import io.jenkins.plugins.casc.ConfiguratorRegistry;
+import io.jenkins.plugins.casc.misc.ConfiguredWithCode;
+import io.jenkins.plugins.casc.misc.JenkinsConfiguredWithCodeRule;
+import io.jenkins.plugins.casc.model.CNode;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import static io.jenkins.plugins.casc.misc.Util.getUnclassifiedRoot;
+import static io.jenkins.plugins.casc.misc.Util.toStringFromYamlFile;
+import static io.jenkins.plugins.casc.misc.Util.toYamlString;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsInstanceOf.instanceOf;
+import static org.junit.Assert.assertThat;
+
+/**
+ * Checks configuration as code integration for local DB.
+ */
+public class ConfigurationAsCodeLocalTest {
+
+    static final String NO_CAUSES_MESSAGE = "No problems were identified. If you know why this problem "
```

Comment:
This looks like the default message, so doesn't really test that the value was correctly set :)

Reply:
```suggestion
    static final String NO_CAUSES_MESSAGE = "No problems were identified. Please contribute failure causes to help others "
```